nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/operations/document/AddPermission.java

```diff
@@ -136,4 +140,12 @@ protected void addPermission(DocumentModel doc) {
         }
     }
 
+    /**
+     * @since 8.10
+     */
+    private boolean isUserFound(String username) {
```

Comment:
There's only two lines in the method and it's only used once, so you can just inline the code at the caller site.